bin/merge

```diff
@@ -49,6 +49,8 @@ if [[ "$merged" == "false" ]]; then
     cd $directory
     git checkout -b $tmp_branch $base_sha
     git pull --ff-only $pr_clone $pr_ref
+    git checkout $base_ref
+    git merge --ff-only --no-ff -m "$(printf "Merge branch '%s'\n\nCloses #%s." "$pr_ref" "$pull_request")" $tmp_branch
```

Comment:
I'm fine with just the auto generated message. If we don't like it down the road we can change it then.